File: Groupe1/TP1/src/controle.c
#include <stdio.h>
//Divisible par 4 mais pas par 6

// int main() {
//     int i;
//     for( i=1; i<=1000; i++) {
//         if(i % 4==0 && i % 6 !=0) {
//             printf("%d ", i);
//         }
//     }
//     printf("\n");

//     return 0;
// }

// Divisible par 8 et pairs

// int main() {
//     int i;
//     for( i=1; i<=1000; i++) {
//         if(i % 2==0 && i % 8==0) {
//             printf("%d ", i);
//         }
//     }
//     printf("\n");

//     return 0;
// }

// Divisibilité par 5 ou 7 mais pas par 10 

int main() {
   
    for (int i = 1; i <= 1000; i++) {
        if ((i % 5 == 0 || i % 7 == 0) && i % 10 != 0) {
            printf("%d ", i);
        }
    }
    printf("\n");
    return 0;
}

File: Groupe1/TP1/src/controle.py
#Divisible par 4 mais pas par 6

print("Nombres divisibles par 4 mais pas par 6 :\n")
for i in range(1, 1001):
    if i % 4 == 0 and i % 6 != 0:
        print(i, end=" ")
print()

#Divisible par 8 et pairs

print("\nNombres pairs et divisibles par 8 :\n")
for i in range(1, 1001):
    if i % 2 == 0 and i % 8 == 0:
        print(i, end=" ")
print()

#Divisibilité par 5 ou 7 mais pas par 10 

print("\nNombres divisibles par 5 ou 7 mais pas par 10 :\n")
for i in range(1, 1001):
    if (i % 5 == 0 or i % 7 == 0) and i % 10 != 0:
        print(i, end=" ")
print()
